pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,51 @@
+{ mkDerivation
+, makeDesktopItem
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  desktopItem = makeDesktopItem {
+    name = "qpaeq";
+    exec = "@out@/bin/qpaeq";
+    icon = "audio-volume-high";
+    desktopName = "qpaeq";
+    genericName = "Audio equalizer";
+    categories = "Music;Sound;";
+    startupNotify = "false";
+  };
+in
+mkDerivation rec {
+  pname = "qpaeq";
+  inherit (pulseaudio) version src;
+
+  buildInputs = [
+    (
+      (python3.withPackages(ps: with ps; [ pyqt5 dbus-python sip ]))
+      .override(args: { ignoreCollisions = true; })
+    )
+  ];
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
+    install -D ./src/utils/qpaeq $out/bin/qpaeq
+    mkdir -p $out/share/applications
+    cp ${desktopItem}/share/applications/*.desktop $out/share/applications
+    '';
```

Comment:
```suggestion
  '';
```

Reply:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`.